GitUI/CommandsDialogs/FormDeleteBranch.cs

```diff
@@ -55,7 +55,7 @@ private void FormDeleteBranchLoad(object sender, EventArgs e)
 
             if (_defaultBranches != null)
             {
-                Branches.SetSelectedText(_defaultBranches.Join(", "));
+                Branches.SetSelectedText(_defaultBranches.Join(" "));
```

Comment:
Should the names be quoted?

Reply:
I don't think so, since branch names can't have spaces in them, right?

Reply:
Leave it now
There are a number of characters that could be interpreted as a space character by the shell but not by Git. At least space and tab is correctly handled
https://git-scm.com/docs/git-check-ref-format

Reply:
Correct.  Have fun with

```bash
git checkout branch name with spaces
```

Now if you want to be evil you could use a non breaking space  (Don't do this.)